docs/keystore.md

```diff
@@ -0,0 +1,18 @@
+## Adapter keystore
+Adapter uses keystore (PKCS #12 **java.security.KeyStore**) for holding certificates and keys which are used for signing http messages (QSEAL) and client tls authentication (QWAC).
+
+By default, a keystore must contain two bags named "default_qwac" and "default_qseal" with an empty password.
```

Comment:
I think it's better to use the same description as in `Pkcs12KeyStore.class` provided - "a keystore is expected to have/include ...". 
In this variant, it sounds like an obligation to have `two bags named "default_qwac" and "default_qseal"`, which is not true as there is a way for a user to provide his/her own aliases for QWAC and QSEAL

Reply:
adjusted